test/unit/spec/Util/jobLogViewerSpec.js

```diff
@@ -7,9 +6,15 @@ define([
     JobLogViewer,
     Runtime
 ) => {
+    'use strict';
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
     describe('Test the job log viewer module', () => {
         let hostNode = null,
             runtimeBus = null;
+
+        beforeAll(() => {
+            window.kbaseRuntime = null;
```

Comment:
Ensure that kbaseRuntime (which holds all the busses and messages and such) is null.
This might get removed in a future PR when ALL specs clean up after themselves. There's no guarantee right now, and this PR's big enough...

Reply:
Could really do with having a global `afterEach` that sets `window.kbaseRuntime` to null, but it would still be possible for state to be contaminated if the tests used more than one level of `describe`s. I can think of a few hacky solutions, but it's better practice for test authors to be careful about ensuring their tests clean up after themselves and don't pollute the global state.

Jasmine's docs don't go into enough detail about state, context, and flow of execution so it's almost inevitable that issues were going to crop up.

Reply:
Right. That's what I've found, too. I'm not sure how to manage that with other test runners, or even jasmine's own test runner, so just being careful here is the best I think I can do for now.